### PR TITLE
I18n

### PR DESCRIPTION
**Pull request by lufaque**

### CODE REVIEW

File: src/components/cards/Card/ErrorCard.tsx [hunk 1]
```diff
@@ -2,13 +2,16 @@ import { Reload } from '@tabler/icons-react'
...
 import { colors } from '../../../constants/colors'
+import {useTranslation} from "react-i18next";
 
 export function ErrorCard({ setIsOk }) {
+  const  {t} = useTranslation()
+}
   return (
     <div className={classNames.container} >
-      <h4 className={classNames.headerTitle}>Couldn’t load data.</h4>
+      <h4 className={classNames.headerTitle}>{t('utils.cuoldNotLoadData')}</h4>
```
Comment (olegkron) [inline src/components/cards/Card/ErrorCard.tsx:12]: couldNotLoadData

File: src/components/cards/ProtocolCard/ProtocolModal/ProtocolModal.tsx [hunk 2]
```diff
@@ -30,64 +31,28 @@ function ChangeTag({ change, period }: { change: number; period: string }) {
...
 		<div className={`card ${classNames.totalContainer} ${classNames.cardContainer}`}>
 			<div className={classNames.priceContainer}>
-				<h4 className="body1">Transactions total</h4>
+				<h4 className="body1">{t('protocolModal.transactionsTotal')}</h4>
 				<h3>${formatNumber(protocol.totalAllTime)}</h3>
 			</div>
 			<div className={classNames.rowContainer}>
-				<ChangeTag change={protocol.change_7d} period="This week" />
-				<ChangeTag change={protocol.change_1m} period="This month" />
+				<ChangeTag change={protocol.change_7d} period={t('protocolModal.thisWeek')} />
```
Comment (olegkron) [inline src/components/cards/ProtocolCard/ProtocolModal/ProtocolModal.tsx:43]: time.thisWeek, time.thisMonth

File: src/components/cards/RewardsCard/RewardsCard.tsx [hunk 3]
```diff
@@ -28,12 +29,15 @@ const RewardsItemCard: FC<RewardsItemCardProps> = ({ name, logoURI, value }) =>
...
+	const { t } = useTranslation()
+
 	if (!stakingState.selectedVault.rewardTokens || !stakingState.selectedVault.rewardTokens.length) {
 		return null
 	}
+
 	return (
 		<div className={classNames.container}>
-			<CardHeader title="Rewards" />
+			<CardHeader title={t('stakingDetailsCard.reward')} />
```
Comment (olegkron) [inline src/components/cards/RewardsCard/RewardsCard.tsx:40]: rewardS

File: src/components/cards/StakingHeaderCard/ManageModal/SelectArea/SelectArea.tsx [hunk 4]
```diff
@@ -59,7 +61,7 @@ export function SelectArea({ selection, direction, dispatch, balance = null, swa
 		<div className={`${classNames.tokenContainer} ${isFocused ? classNames.inputFocused : ''}`} onClick={() => handleAreaClick(inputRef)}>
 			<div className={classNames.tokenRow}>
 				<div className={classNames.tokenRowHeader}>
-					<p>{capitalize(direction)}</p>
+					<p>{t(`tokenArea.${direction}`)}</p>
```
Comment (olegkron) [inline src/components/cards/StakingHeaderCard/ManageModal/SelectArea/SelectArea.tsx:64]: We also should use capitalise here

File: src/components/cards/SwapCard/SwapProgress/ContactSupport/ContactSupport.tsx [hunk 5]
```diff
@@ -36,33 +38,33 @@ export function ContactSupport({ swapState, swapDispatch }: ContactSupportProps)
 	return (
 		<div className={classNames.container}>
 			<div>
-				<p className={'body1'}>We apologise that you had issues with your transaction. We will do our best to resolve the issue.</p>
+				<p className={'body1'}>{t('contactSupportCard.apologise')}</p>
```
Comment (olegkron) [inline src/components/cards/SwapCard/SwapProgress/ContactSupport/ContactSupport.tsx:41]: txFailedMessage

**Review by olegkron — CHANGES_REQUESTED** [review]

File: src/components/cards/StakingHeaderCard/ManageModal/SelectArea/SelectArea.tsx (hunk 4 above)
Comment (lufaque) [reply src/components/cards/StakingHeaderCard/ManageModal/SelectArea/SelectArea.tsx:64]: this text has already been capitalized

**Pull request merged**